src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestUserReceiptHandler.java

```diff
@@ -74,38 +74,56 @@ public Mono<BaseTransactionEvent<?>> handle(TransactionAddUserReceiptCommand com
                 );
 
         Mono<it.pagopa.ecommerce.commons.domain.v2.TransactionClosed> alreadyProcessedError = transaction
```

Comment:
Just food for thought, maybe better for another PR: given that this chain is quite long and filled with conditionals, what do you think about making a method that computes the result?

Reply:
I'll open an issue to track this